trackma/utils.py

```diff
@@ -54,7 +85,7 @@
 SEASON_SPRING = 2
 SEASON_SUMMER = 3
 SEASON_FALL = 4
-
+SEASON=[ 'All','Winter','Spring','Summer','Fall' ]
```

Comment:
```suggestion
SEASON = ['All', 'Winter', 'Spring', 'Summer', 'Fall']
```

Also, where is this used?

Reply:
We can use them to show String instead of a Number.